lib/lita/handlers/karma/chat.rb

```diff
@@ -104,7 +104,7 @@ def delete(response)
 
     def define_dynamic_routes(pattern)
       self.class.route(
-        %r{(#{pattern})\+\+#{token_terminator.source}},
+        %r{(#{pattern}) {0,1}\+\+#{token_terminator.source}},
```

Comment:
The regex hacker in me wonders if the space should be \s to allow any whitespace, but would include newlines, which means someone pasting patch (diff -u) format might get errant karma modifications.

 I think I’m over-engineering it. Let’s just start with spaces.